src/test/FuzzerImpl.cpp

```diff
@@ -1191,8 +1191,8 @@ std::array<TrustLineParameters, 12> constexpr trustLineParameters{
 
      // these 5 trustlines are required for claimable balances
      {2, AssetID(4), 256, 256},
-     {3, AssetID(4), 256, 256},
-     TrustLineParameters::withAllowTrust(4, AssetID(3), 256, 256,
+     {3, AssetID(4), INT64_MAX, 50},
```

Comment:
The trustline balance specified here is treated as a max value. If the balance after setup is under this value, it is not topped off. So in this case, the actual balance is 50, with an available limit of 100.

I don't think this was the intended behavior for setup though. The trustline should be topped off since the use is expected a certain balance for the trustline to end with. I added a commit where we top off the trustline to `mAssetAvailableForTestActivity` if we can.

Reply:
Upon reviewing this again, I see my misunderstanding. The `INT64_MAX` is a pre-setup value, but `INT64_MAX - 50` will be deposited into a liquidity pool. I agree that topping off the trustline probably makes sense, and it was missing because this was originally designed for setup with high balances but testing with low balances (so topping off was unnecessary).

I think this implementation of topping off is missing a few things, specifically it doesn't handle native asset. But I'm comfortable merging this and handling that in another PR.